schemas/livePaperVersion.schema.tpl.json

```diff
@@ -0,0 +1,59 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LivePaperVersion",
+  "_extends": "/core/products/researchProductVersion.schema.tpl.json",
+  "required": ["digitalIdentifier", "license"],
+  "properties": {
+    "about": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add links to the datasets, models or software contained within or described by this version of the live paper.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DatasetVersion",
+        "https://openminds.ebrains.eu/core/ModelVersion",
+        "https://openminds.ebrains.eu/core/SoftwareVersion"
+      ]
+    },
+    "abstract": {
```

Comment:
a ResearchProductVersion already has an abstract field called "description"

Reply:
ok, I've removed this